modules/chapter2/pages/git-auth.adoc

```diff
@@ -0,0 +1,226 @@
+----
+    enabled:
+      github: true
+      githubOrg: true
+    *catalog:
+      providers:
+        githubOrg:
+          default:
+            id: development
+            orgUrl: ${GITHUB_ORG_URL}*
+----
+
+. Click `Save`.
+
+### Upgrade RHDH Helm Chart
+
+The final step is to inject the secret you created with the GitHub App credentials into the RHDH container. You do this by editing the helm chart configuration and refer to the `rhdh-secrets`. You will also enable the `GitHub Organization discovery` dynamic plugin.
+
+. Edit the `rhdh` helm chart, toggle `Form view`, and expand `Root Schema > global > Dynamic plugins configuration > List of dynamic plugins that should be installed in the backstage application`. Add the value `./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org-dynamic` to the `Package specification...` field. This value enables the dynamic plugin for GitHub Organization discovery.
++
+image::9-gh-auth.png[title=Enable the GitHub Org Discovery Dynamic Plugin]
+
+. Next, expand `Root Schema > Backstage chart schema > Backstage parameters > Backstage container environment variables from existing Secrets` and add `rhdh-secrets` to refer to the secret you created earlier.
++
+image::3-gh-auth.png[title=Reference the Secret]
+
+. Click `Upgrade`.
+
+. Wait for a few minutes while the helm chart is upgraded, and the RHDH pod is re-created with the new configuration.
+
+### Verify User Sign In
+
+. If you are signed in as a `Guest` RHDH user, then sign out by selecting the `Sign Out` (expand the `...` next to the Profile card) in the `Settings > Profile` page.
++
+image::guest-sign-out.png[title=Guest Sign out,width=600]
+
+. Sign in using `GitHub` sign-in method in the RHDH home page
++
+image::sign-in-github.png[title=GitHub Sign in,width=600]
+
+. You will be re-directed to a GitHub prompt asking you to sign in using your GitHub credentials. Once you are signed in, you will be asked to allow access to your GitHub profile data.
++
+image::4-gh-auth.png[title=GitHub Allow Access Prompt,width=400]
+
+. After you are signed in, navigate to the `Settings` page. Notice that GitHub has authenticated the user and provided identity information to RHDH.
++
+image::gh-identity-visible.png[title=GitHub Identity in Settings Page,width=500]
+
+. Click on the logged in `User Entity` in the `Settings` page to view the detail imported from GitHub.
++
+image::view-gh-profile.png[title=View Profile Data,width=400]
+
+. Click on the RHDH `Catalog` in the sidebar. Select the `Kind` drop-down, and note that two new entities called `User` and `Group` are now visible.
+. Select `Group` in the `Kind` drop-down to view the teams (mapped into RHDH Groups) imported from the GitHub organization.
++
+image::view-kind-groups.png[title=View Groups (Teams) from GitHub]
+
+. Select `User` in the `Kind` drop-down to view the users imported from the GitHub organization.
++
+image::view-kind-users.png[title=View Users Imported from GitHub]
```

Comment:
noted. let's make a little assumption that GitHub org admins know how to do this to avoid polluting the page with too many screenshots. I'll add a reference to GH docs